Building of Docker image using root Dockerfile fails

**Expected behaviour**
Docker build pass

**Steps to reproduce** 

- clone grafana repo from master
- enter into main dir
- run docker build -t blabla .


**Error at:**

```
Step 33/37 : COPY --from=1 /usr/src/app/tools ./tools
COPY failed: stat /var/lib/docker/overlay2/bf3d606b1e9ccc9505d9c154ec2df3b9763293b447c8126ba058d081d0f8d694/merged/usr/src/app/tools: no such file or directory
Makefile:135: recipe for target 'build-docker-full' failed
make: *** [build-docker-full] Error 1
```


I've put a command at last step of building second image in Dockerfile and i don't see any **tools** directory.

**ootuput:**


`Step 21/37 : RUN ls
 ---> Running in 2b109cda33c7
Gruntfile.js
emails
node_modules
package.json
packages
public
scripts
tsconfig.json
yarn.lock
`

--- a/Dockerfile
+++ b/Dockerfile
@@ -25,7 +25,7 @@ COPY packages packages
 
 RUN yarn install --pure-lockfile --no-progress
 
-COPY Gruntfile.js tsconfig.json .eslintrc .editorconfig .browserslistrc ./
+COPY Gruntfile.js tsconfig.json .eslintrc .editorconfig .browserslistrc .prettierrc.js ./
 COPY public public
 COPY scripts scripts
 COPY emails emails

